src/main/python/powerFactory2json/pf2json.py

```diff
@@ -0,0 +1,3153 @@
+import powerfactory  # @UnresolvedImport @UnusedVariable
+import json
+import inspect
+import os
+
+#################
+# Configuration #
+#################
+
+exported_grid_dir = "../out"
```

Comment:
`../out/` is the default intelliJ build/compile dir. I would therefore prefer something like 
```suggestion
exported_grid_dir = "../pfGridExport"
```